blackbox/harness/deps.py

```
@@ -86,32 +107,138 @@ class Drive9DependencyManager(DependencyManager):
+        """Map logical/Debian package names to the active package manager."""
+        if manager == "apt":
+            # Suites already pass Debian/Ubuntu names.
+            return list(dict.fromkeys(packages))
+        mapped: list[str] = []
+        for package in packages:
+            mapped.append(_DEBIAN_TO_ARCH.get(package, package))
+        return list(dict.fromkeys(mapped))
+
+    def _package_installed(self, manager: PackageManager, package: str) -> bool:
+        if manager == "apt":
+            proc = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)